solutions/usaco-896.mdx

```diff
@@ -135,6 +135,7 @@ with open('mountains.in') as read:
 		a, b = map(int, read.readline().split())
 		pos.append((a - b, a + b))
 
+# sort mountain positions 
```

Comment:
...well of course you're sorting positions
what are you sorting them _by_?

Reply:
okay. tbh i forgot what this problem was about i'll resolve and get back to u